app/views/embedded_terraform_template/show.html.haml

```diff
@@ -0,0 +1,3 @@
+#main_div
+  = render :partial => 'layouts/textual_groups_generic'
+  = react('WorkflowPayload', {:recordId => params[:id].to_s})
```

Comment:
@GilbertCherrie do we need an `EmbeddedTerraformTemplatePayload` here?

Reply:
If the payload is the same as the workflow payload then we can leave it as is. If the format of the payload is different then we could create a new react component called `EmbeddedTerraformTemplatePayload`.

Reply:
The format of the payload will be different

Reply:
Okay I've just removed this for now, once we figure out what the payload will look like we can revisit this.